exec/java-exec/src/main/java/org/apache/drill/exec/physical/resultSet/impl/ContainerState.java

```diff
@@ -172,4 +172,8 @@ public void close() {
       colState.close();
     }
   }
+
+  public boolean isDict() {
+    return false;
+  }
```

Comment:
Should clients of this class care what kind of container this is? Shouldn't all DICT-specific code be in the DICT related classes?

Reply:
It is better, this was removed, yes. I'll try to eliminate this.